codersdk/agentsdk/aws.go

```diff
@@ -0,0 +1,97 @@
+package agentsdk
```

Comment:
Suggestion (optional): Consider moving into sub-package, assuming there aren't any blocking dependencies to this package that would cause a circular import.